### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Andrea**
add a rules of engagement section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# Rules of engagement
+
+* When a PR is open the approver (should/ shouldn't) merge
```
Comment (zainab-ali) [inline README.md]: I'd go with `should`, or we might end up with approved PRs gathering dust

File: README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+* ?? other rules :D
```
Comment (zainab-ali) [inline README.md]: Let's have a think about this one

File: README.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+# Rules of engagement
```
Comment (zainab-ali) [inline README.md]: It might read better if we put this section below "Project suggestions" instead of above

File: README.md (hunk 2 above)
Comment (Andrea) [reply README.md]: I removed it

**Comment by DanielaSfregola** [comment]
LGTM

**Pull request merged**